images/promtail/main.tf

```diff
@@ -0,0 +1,54 @@
+terraform {
+  required_providers {
+    apko = { source = "chainguard-dev/apko" }
+  }
+}
+
+variable "target_repository" {
+  description = "The docker repo into which the image and attestations should be published."
+}
+
+module "latest" {
+  source = "../../tflib/publisher"
+
+  name = basename(path.module)
+
+  target_repository = var.target_repository
+  config            = file("${path.module}/configs/latest.apko.yaml")
+}
+
+module "dev" { source = "../../tflib/dev-subvariant" }
+
+module "latest-dev" {
+  source = "../../tflib/publisher"
+
+  name = basename(path.module)
+
+  target_repository = var.target_repository
+  # Make the dev variant an explicit extension of the
+  # locked original.
+  config         = jsonencode(module.latest.config)
+  extra_packages = module.dev.extra_packages
+}
+
+module "version-tags" {
+  source  = "../../tflib/version-tags"
+  package = "promtail"
+  config  = module.latest.config
+}
+
+module "tagger" {
+  source = "../../tflib/tagger"
+
+  depends_on = [module.test-latest]
+
+  tags = merge(
+    { for t in toset(concat(["latest"], module.version-tags.tag_list)) : t => module.latest.image_ref },
+    { for t in toset(concat(["latest"], module.version-tags.tag_list)) : "${t}-dev" => module.latest-dev.image_ref },
```

Comment:
```suggestion
    { for t in toset(concat(["latest-dev"], module.version-tags.tag_list)) : "${t}-dev" => module.latest-dev.image_ref },
```

Reply:
Are you sure? All the other images have the same setup...

https://github.com/chainguard-images/images/blob/main/images/argocd/main.tf#L65
https://github.com/chainguard-images/images/blob/main/images/nginx/main.tf#L52
https://github.com/chainguard-images/images/blob/main/images/buck2/main.tf#L52

Reply:
I was wrong, sorry.